### PR TITLE
Hypen in GtHub Repo URL

### PR DESCRIPTION
**Pull request by nokeeo**

### CODE REVIEW

**Comment by sam1360** [comment]
Looks good to me! 👍

**Comment by Ourobor** [comment]
Looks good to me (y)

File: sourceHandlers/gitHub.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const SourceHandler = require('./sourceHandler');
 const baseUrl = 'https://api.github.com/';
 const userAgent = 'SoftwareCitationCore';
 
-const urlRegex = /^github\.com\/(\w+)\/(\w+)/;
+const urlRegex = /^github\.com\/([\w,\-]+)\/([\w,\-,\_]+)/;
```
Comment (faokryn) [inline sourceHandlers/gitHub.js]: Doesn't `\w` match underscores?

Comment (nokeeo) [reply sourceHandlers/gitHub.js]: Ah it does.  Will fix
http://www.fileformat.info/info/unicode/category/Pc/list.htm

**Comment by faokryn** [comment]
👍

**Pull request merged**